lightbar_manager/src/lightbar_manager_worker.cpp

```diff
@@ -121,7 +127,7 @@ namespace lightbar_manager
 
         if (raw_map.size() < 4)
```

Comment:
Why did the input to this function change? Wont this map always be <4, we're only assigning it one value

Reply:
- ROS2 doesn't have dictionary parameter loading, so I turned it into 2 vectors of 1-to-1 correlation (just like how carma_wm_ctrl loads the offset parameters).
 - Yes we are assigning only 1 element, but there has to be at least 4 elements (or key/value pairs) total in the map, since there is minimum 4 CDATypes that needs to be mapped.